test/bye_bye_bye/utils_test.exs

```diff
@@ -101,7 +101,7 @@ defmodule ByeByeBye.UtilsTest do
   describe "build_cancellation_entity/2" do
     test "creates a feed entity with proper cancellation details" do
       trip_id = "123"
-      now = 1_706_000_000
```

Comment:
why was this unix time before?

Reply:
This function originally only needed `now` for the `timestamp` field which is in unix. Now that I need it as a `DateTime` too, I changed the type of the parameter and converted it in the function body.